service/src/main/java/bio/terra/workspace/app/controller/ControlledAzureResourceApiController.java

```diff
@@ -137,6 +126,13 @@ public ControlledAzureResourceApiController(
     return new ResponseEntity<>(response, HttpStatus.OK);
   }
 
+  @Override
+  public ResponseEntity<ApiCreatedControlledAzureNetwork> createAzureNetwork(
+      UUID workspaceId, ApiCreateControlledAzureNetworkRequestBody body) {
+    final AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
+    return ControlledAzureResourceApi.super.createAzureNetwork(workspaceId, body);
```

Comment:
I believe this method should be constructing the AzureNetwork controlled resource, and call the `controlledResourceService.createNetwork()` method. See the `createAzureDisk` method above for an example

Reply:
Yes. The API layer is supposed to convert the API interface types such as `ApiCreatedControlledAzureNetworkRequestBody` into internal types. There are a few exceptions, but this does not need to be one of them.

Reply:
updated to construct the resource and call `controlledResourceService.createNetwork()`